src/transaction/log_tran_table.c

```diff
@@ -3541,25 +3541,15 @@ logtb_tran_update_unique_stats (THREAD_ENTRY * thread_p, const BTID * btid, int
   if (write_to_log)
     {
       /* log statistics */
-      char undo_rec_buf[3 * OR_INT_SIZE + OR_BTID_ALIGNED_SIZE + MAX_ALIGNMENT];
-      char redo_rec_buf[3 * OR_INT_SIZE + OR_BTID_ALIGNED_SIZE + MAX_ALIGNMENT];
-      RECDES undo_rec, redo_rec;
+      constexpr size_t data_size (OR_BTID_ALIGNED_SIZE + (3 * OR_INT_SIZE));
```

Comment:
- Constants are named using uppercase characters. 
- Use assignment instead of constructor to initialize primitive types.
```suggestion
      constexpr size_t DATA_SIZE = (OR_BTID_ALIGNED_SIZE + (3 * OR_INT_SIZE));
```